drivers/SmartThings/matter-switch/profiles/matter-bridge.yml

```diff
@@ -1,4 +1,4 @@
-name: bridge
+name: matter-bridge
```

Comment:
I would think this profile is definitely being used in the field, so I don't think we can change the name safely